lib/services/welcome_screen_items.dart

```diff
@@ -21,16 +21,35 @@ List<WelcomeScreenItem> getWelcomeScreenItems(BuildContext context) {
         description: AppLocalizations.of(context)!.welcome_view_foss_warn_text,
         imagePath: "assets/app_icon/app_icon.png"),
     WelcomeScreenItem(
-        title: "FOSS Public Alert Server", //@todo translate
+        // @todo translate welcome_view_foss_public_alert_server_headline
```

Comment:
I recommend marking todo's in the format of `// TODO(your name): some text describing the todo`. That's the official Flutter way of doing it and it allows IDE's like VSCode to list it in the "problem" tab (alongside warnings and errors), making sure it's not forgotten about.